examples/simple/setup.py

```diff
@@ -34,11 +34,14 @@ def add_data_files(path):
     version = VERSION,
     description = 'Jupyter Server Example',
     long_description = open('README.md').read(),
-    python_requires = '>=3.5',
+    python_requires = '>=3.6',
```

Comment:
:skull_and_crossbones:

Reply:
I guess I'm not following the skull & crossbones commentary here, sorry about that.  The change seems fine.

Reply:
Just meant 3.5 is EOL!